submodules/power_cloudconnection_create/variables.tf

```diff
@@ -18,6 +18,13 @@ variable "transit_gateway_name" {
   type        = string
 }
 
+
+variable "cloud_connection_name_prefix" {
+  description = "Optional prefix for cloud connection name. If null default cloud connection name will be <zone>-conn-1. Else it <prefix>-<zone>-conn-1"
```

Comment:
`If null default cloud connection name will be <zone>-conn-1` -> `If null or empty string, default cloud connection name will be <zone>-conn-1.`